server/metrics.py

```diff
@@ -1,21 +1,53 @@
 """
 Prometheus metric definitions
 """
+from enum import Enum, unique
 
 from prometheus_client import Counter, Gauge, Histogram, Info
 
+
+@unique
+class MatchLaunch(Enum):
```

Comment:
The issue with Enum.value would also be solved if you just not make this an Enum. Then it will just be a class for holding some string constants like the RatingType class.